<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlasmoidTasksConfig</class>
 <widget class="QWidget" name="PlasmoidTasksConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>418</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="unlockLabel">
     <property name="text">
      <string>Widgets are currently locked. To change which widgets are shown, please unlock the widgets first.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="unlockButton">
     <property name="text">
      <string>Unlock widgets</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="KCategorizedView" name="visibleItemsView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KCategorizedView</class>
   <extends>QListView</extends>
   <header location="global">kcategorizedview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
